src/disc/omegah/OmegahConnManager.cpp

```diff
@@ -449,8 +470,9 @@ OmegahConnManager::getAssociatedNeighbors(const LO& /* el */) const
 }
 
 // Where element ielem start in the 1d connectivity array
-int OmegahConnManager::getConnectivityStart (const LO ielem) const {
-  return 0;
+int OmegahConnManager::getConnectivityStart (const LO localElmtId) const {
+  static_assert(sizeof(Omega_h::GO) == sizeof(GO));
```

Comment:
Looking at this again, I think this `static_assert` is not needed here.  A static assert was added at [1] where global ids are passing from Omega_h to Albany via the connectivity array and problems would occur if the types didn't match.  It doesn't matter here as we are simply passing back an index into the connectivity array.

Are you OK with leaving the `static_assert` at [1] or do you think it would be better to move it somewhere else in the header?

[1] https://github.com/SCOREC/Albany/blob/f9bba7053b5e911f3e3889238c5095e4a72b9b6b/src/disc/omegah/OmegahConnManager.hpp#L102-L106

Reply:
I think it's better if it is at the top of the class, in class scope. I think it's preferable to see any static assumption (that has broad impacts) at the top of the class/file, rather than it being buried inside a function...